### ISSUE TITLE
Integrating ACT Rules into WCAG Supporting Documents

### ISSUE THREAD

**Issue by nitedog** [w3c/wcag-act#446]
This issue continues one aspect of issue #353 "Process proposal: Publishing ACT Rules on w3.org/WAI" -- the integration of these published ACT Rules as part of the WCAG Supporting Document (currently Understanding WCAG 2 and Techniques for WCAG 2):

> 1. New pages, one for each rule

Here is an example of a published ACT Rule "HTML page has title":
 - https://www.w3.org/WAI/standards-guidelines/act/rules/html-page-has-title-2779a5/

> 2. A new WCAG Test Rules page (similar to Techniques for WCAG https://www.w3.org/WAI/WCAG21/Techniques/), showing a list of all WCAG test rules.

Currently listed here but may eventually get an own listing:
 - https://www.w3.org/WAI/standards-guidelines/act/#act-rules-repository

> 3. Links in Understanding WCAG documents, for rules that map to success criteria, similar to how techniques are linked from the understanding documents.

As an illustrative example, let's use "Understanding Success Criterion 2.4.2: Page Titled":
 - https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html

The proposal would be to add a new heading level 2 sub-section directly after "Techniques":

> # ACT Rules
> The following Accessibility Conformance Testing (ACT) Rules could be used by automated testing tools and manual testing methodologies, to help identify potential failures. For information on using other ACT Rules, see Understanding ACT Rules for WCAG Success Criteria.
> - [HTML page has title](https://www.w3.org/WAI/standards-guidelines/act/rules/html-page-has-title-2779a5/)

**Note:** the "Understanding ACT Rules for WCAG Success Criteria" would be similar to [Understanding Techniques for WCAG Success Criteria](https://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques) only much shorter and reusing existing wording from the [ACT Rules Overview](https://www.w3.org/WAI/standards-guidelines/act/) page. This would address issue #180 too.

> 4. Links from WCAG Techniques, similar to how rules have links to rules in its "Background" section

Staying with the example of "HTML page has title" from above, the proposal would be to add a new heading level 2 sub-section directly after "Related Techniques":

> # Related ACT Rules
> - [HTML page has title](https://www.w3.org/WAI/standards-guidelines/act/rules/html-page-has-title-2779a5/)

This would be added to the following Techniques and Failures at least:
 - [H25: Providing a title using the title element](https://www.w3.org/WAI/WCAG21/Techniques/html/H25)
- [F25: Failure of Success Criterion 2.4.2 due to the title of a Web page not identifying the contents](https://www.w3.org/WAI/WCAG21/Techniques/failures/F25)

> 5. Inclusion in the How to Meet WCAG 2 quickref, by adding a new section under each success criterion called "Test Rules"

This would need revision of the underlying JSON data, parsing, and potentially some of the UI elements (eg. filter switches), and will be pursued in a later phase.
